diff --git a/breathe/renderer/sphinxrenderer.py b/breathe/renderer/sphinxrenderer.py
index 93aad82..4382c8f 100644
--- a/breathe/renderer/sphinxrenderer.py
+++ b/breathe/renderer/sphinxrenderer.py
@@ -1354,8 +1354,8 @@ class SphinxRenderer:
         '''
         section = nodes.section()
         section['ids'].append(self.get_refid(node.id))
-        section += self.create_doxygen_target(node)
         section += nodes.title(node.title, node.title)
+        section += self.create_doxygen_target(node)
         section += self.render_iterable(node.content_)
         return [section]
 
